templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -488,12 +497,11 @@ Parameters:
     Default: ""
     Type: String
     Description: Named Key Pair name to use with this repository. The key should be imported into the SSM parameter store. (Optional)
-  CloudWatchIntegration:
-    Default: "Metrics and Logs"
+  ExportPrefix:
+    Default: 'ATL-'
+    Description:
+      Prefix for the exported variables (VPCID, SubnetIDs, KeyName) that are reused by other Atlassian AWS Quickstarts.
```

Comment:
@ddomingorh this will require documentation update. It would be also good review the wording.

Reply:
Can we change this to something like:

> Prefix used in all variables exported from this deployment’s Atlassian Standard Infrastructure (VPCID, SubnetIDs, KeyName). Use different prefixes to deploy multiple Atlassian Standard Infrastructures in the same AWS region.